.travis.yml

```diff
@@ -15,8 +15,8 @@ before_script:
 - sleep 3
 - lerna run --scope @requestnetwork/request-network.js testdeploy
 script:
-- lerna run --scope @requestnetwork/request-network.js test
-- lerna run --scope request-network-smart-contracts test
+- travis_wait lerna run --scope @requestnetwork/request-network.js test --stream
```

Comment:
Do you mind removing the travis_wait? It didn't solve the issue.
We can keep the additions of --stream, though